.github/release-drafter.yml

```diff
@@ -0,0 +1,37 @@
+# https://github.com/release-drafter/release-drafter?tab=readme-ov-file#configuration-options
+template: |
+  $CHANGES
+
+  **Full Changelog**: https://github.com/$OWNER/$REPOSITORY/compare/$PREVIOUS_TAG...v$RESOLVED_VERSION
+name-template: 'docs-builder v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+change-template: '- $TITLE by @$AUTHOR in #$NUMBER'
+categories:
+  - title: '💥 Breaking Changes'
+    labels:
+      - 'changelog:breaking' # When a breaking change is introduced
+  - title: '✨ Features'
+    labels:
+      - 'changelog:feature' # When a new feature is introduced
+      - 'changelog:enhancement' # When an existing feature is improved
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'changelog:fix' # When a bug is fixed
```

Comment:
can we reuse `bug` for this?

Reply:
I'm always hesitant about this.

I understand that an issue has the "bug" label.

But I'm always unsure about using the same label on a PR because it's actually the fix,  isn't it?

😅 

WDYT?

Reply:
we can support both, I guess